src/subgraph/actions/listBuyNowSales.ts

```diff
@@ -21,14 +22,17 @@ export const listBuyNowSales = async <T>(
   }
 
   const buys: TokenBuy[] = queryResult.data.domainTokenSolds.map((e) => {
-    return {
+    const buy: TokenBuy = {
       tokenId: e.tokenId,
       contract: e.contractAddress,
       amount: e.amount,
       seller: e.seller.id,
       buyer: e.buyer.id,
-      timestamp: e.timestamp
-    } as TokenBuy;
+      timestamp: e.timestamp,
```

Comment:
This is read right from the chain event so yes it would be in seconds but I just checked to be sure and there is conversion downstream to make this work as expected so better to leave it unchanged